vllm_ascend/worker/worker_v1.py

```
@@ -133,6 +135,16 @@ def init_device(self):
 
         # Initialize the distributed environment.
         self._init_worker_distributed_environment()
+        # Bind cpu
+        if ascend_envs.VLLM_ASCEND_CPU_BINDING:
+            try:
+                bind_cpus(self.local_rank, ratio=1.0)
+            except RuntimeError as e:
+                logger.error(f"{e} in {self.local_rank}")
+            except ValueError as e:
+                logger.error(f"{e} in {self.local_rank}")
+            except Exception:
+                logger.info("Skip binding cpu.")
```

We should report the error, either by raising an exception or logging it, to inform the user.

---

OK，I will modify it.